services/src/main/resources/db/changelog/gamification.db.changelog-1.0.0.xml

```diff
@@ -697,4 +697,20 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
       </addColumn>
     </changeSet>
 
+    <changeSet author="exo-gamification" id="1.0.0-63">
+      <createTable tableName="GAMIFICATION_CONNECTOR_ACCOUNTS">
+        <column name="ID" type="BIGINT" autoIncrement="${autoIncrement}" startWith="1">
+          <constraints nullable="false" primaryKey="true" primaryKeyName="PK_GAMIFICATION_CONNECTOR_ACCOUNTS"/>
+        </column>
+        <column name="CONNECTOR_NAME" type="NVARCHAR(250)">
+          <constraints nullable="false"/>
+        </column>
+        <column name="REMOTE_ID" type="NVARCHAR(250)"/>
+        <column name="USER_ID" type="BIGINT">
+          <constraints nullable="false"/>
+        </column>
+      </createTable>
+      <addUniqueConstraint columnNames="CONNECTOR_NAME, REMOTE_ID" tableName="GAMIFICATION_CONNECTOR_ACCOUNTS"/>
+      <createSequence sequenceName="SEQ_GAMIFICATION_CONNECTOR_ACCOUNTS_ID" startValue="1" />
```

Comment:
```suggestion
      <createSequence sequenceName="SEQ_GAMIFICATION_CONNECTOR_ACCOUNTS_ID" startValue="1" />
```

Please add this into a dedicated changeSet in order to apply it on some Rdbms only, as it's not useful for MySQL by example.